teammapper-backend/src/map/services/maps.service.spec.ts

```diff
@@ -7,19 +7,21 @@ import { MmpNode } from '../entities/mmpNode.entity';
 import { Repository } from 'typeorm';
 import { ConfigModule } from '@nestjs/config';
 import AppModule from '../../app.module';
-import { createTestConfiguration } from '../../../test/db';
+import { createTestConfiguration, destroyWorkerDatabase } from '../../../test/db';
 import { mapMmpNodeToClient } from '../utils/clientServerMapping';
+import { truncateDatabase } from 'test/helper';
 
 describe('MapsController', () => {
   let mapsService: MapsService;
   let nodesRepo: Repository<MmpNode>;
   let mapsRepo: Repository<MmpMap>;
+  let moduleFixture: TestingModule;
 
   beforeAll(async () => {
-    const moduleFixture: TestingModule = await Test.createTestingModule({
+    moduleFixture = await Test.createTestingModule({
       imports: [
         ConfigModule,
-        TypeOrmModule.forRoot(createTestConfiguration()),
+        TypeOrmModule.forRoot(await createTestConfiguration(process.env.JEST_WORKER_ID)),
```

Comment:
where is the env variable coming from, do we have to set this in the environment or is the framework managing the id?